src/app/controllers/user.controllers.ts

```diff
@@ -1,19 +1,33 @@
 import { Request, Response } from "express";
-import { findUsersCount } from "../repositories/users.repository";
+import {
+  findUser,
+  findUsersCount,
+  processingUserList,
+} from "../repositories/users.repository";
 import {
   loginUser,
   registerUser,
   removeMe,
   changePassword,
   paginate,
+  sendVerifyEmail,
+  verifyAccount,
 } from "../../services/user.service";
+
 export const registationController = async (req: Request, res: Response) => {
   const userEmail = req.body.email;
   const userPassword = req.body.password;
-  if (registerUser(userPassword, userEmail)) {
-    res.status(200).json({ message: "registrated" });
+  const candidateEmail = await findUser({ email: userEmail });
+  if (!candidateEmail) {
```

Comment:
if (candidateEmail) {
return res.status(400).json({ message: "user with this  email already exist" });
}
...